pod/perlclass.pod

```diff
@@ -4,7 +4,7 @@ perlclass - Perl class syntax reference
 
 =head1 SYNOPSIS
 
-    use v5.36;
+    use v5.38;
```

Comment:
Correct me if I'm wrong, but I think that line might as well be totally removed since the `class` keyword enables subroutine signatures by default.